raiden-ts/src/raiden.ts

```diff
@@ -89,17 +86,35 @@ export class Raiden {
    */
   public readonly events$: Observable<RaidenEvent>;
 
+  /**
+   * Observable of completed and pending transfers
+   * Every time a transfer state is updated, it's emitted here. 'secrethash' property is unique and
+   * may be used as identifier to know which transfer got updated.
+   */
+  public readonly transfers$: Observable<RaidenSentTransfer>;
+
   /**
    * Expose ether's Provider.resolveName for ENS support
    */
   public readonly resolveName: (name: string) => Promise<Address>;
 
   /**
-   * Observable of completed and pending transfers
-   * Every time a transfer state is updated, it's emitted here. 'secrethash' property is unique and
-   * may be used as identifier to know which transfer got updated.
+   * Get token information: totalSupply, decimals, name and symbol
+   * Rejects only if 'token' contract doesn't define totalSupply and decimals methods.
+   * name and symbol may be undefined, as they aren't actually part of ERC20 standard, although
+   * very common and defined on most token contracts.
+   *
+   * @param token  address to fetch info from
+   * @returns  token info
    */
-  public readonly transfers$: Observable<RaidenSentTransfer>;
+  public getTokenInfo: (
+    token: string,
+  ) => Promise<{
```

Comment:
In `raiden-ts/src/types.ts` we have `TokenInfo` for this

Reply:
Trivial/simple type declarations, which are used only once, don't need to have its own export. There's plenty of inline types throughout the code. As the type itself is self-describing, removing it is cleaning the type namespace, the public interface and one explicit schema for us to maintain.